server/tests/test_app.py

```diff
@@ -6,11 +7,49 @@
 
 from server.app import app  # Import the Flask app
 
+# Helper function to generate private and public keys for testing
+def generate_keys(path):
+    print(path)
+    from cryptography.hazmat.primitives import serialization
```

Comment:
Duplicated import?

Reply:
Thanks, interesting Ruff and isort say nothing. I cleaned up the cruft.